### PR TITLE
SISRP-7707 - Resize Find button in UC Planner grid

### PR DESCRIPTION
**Pull request by darrylumich**
https://jira.berkeley.edu/browse/SISRP-7707

### CODE REVIEW

**Comment by christianvuerings** [comment]
@darrylumich could you have a look at the build failures at https://travis-ci.org/ucberkeley/sis-custom/builds/81430141 ?

File: src/css/cs/items/_aa_planner.scss [hunk 1]
```diff
@@ -16,6 +16,19 @@
   .uc-fluid-table table {
     font-size: 11px;
   }
+  
+  .ps_content .uc-fluid-table table table .ps_box-button a {
```
Comment (christianvuerings) [inline src/css/cs/items/_aa_planner.scss]: @darrylumich does it work when you nest this under `.uc-fluid-table table`?

Something like:

``` scss
.uc-fluid-table table {
  font-size: 11px;
  table .ps_box-button a {
    background: ...
  }
}
```

**Comment by darrylumich** [comment]
Okay, I can remove all the color references, as they are superfluous.  Also, will make the 0px reference simply 0.

It appears I can't combine these change with the one for font-size: 11px  That has a trail like so:\

.uc-planner .uc-fluid-table table

While the new code has one extra section, ps_content:

.uc-planner .ps_content .uc-fluid-table table ....

**Comment by christianvuerings** [comment]
@darrylumich could you have a look at the last build failure?
Also make sure to ping me as soon as Travis is green by typing "@christianv" in a comment on GitHub.

**Pull request merged**